### DISCUSSION TITLE
DS2408, DS2762, DS18b20, DS2413, DS2438, DS2450

### DISCUSSION THREAD

**Discussion by garikhgh** [mattjlewis/diozero#155] [General]
Hi. I have a program written in python that runs a soft on Raspberry Pi. Now I want to rewrite the soft using Java and I found this lib. So I could not find driver implementations for the following devices DS2408, DS2762, DS18b20, DS2413, DS2438, DS2450

**Comment by mattjlewis** [comment]
Hey there. I’ve tested with a DS18b20 based on having the Linux 1-wire driver loaded. See the W1Device class - a bad name I know, will be renaming it in the next release.
I’ve just added support via Firmata too.
